### ISSUE TITLE
Wrong passanger count when player cancel entering task

### ISSUE THREAD

**Issue by TuK4z** [altmp/altv-issues#2288]
### Description of the problem

When player trying to enter vehicle and press walking buttons to cancel entering task - vehicle have wrong passanger count. If player enter and leave vehicle it resets and passanger count is correct again.

### Reproduction steps

Try enter vehicle >> Cancel entering task by prasing any walking buttons >> check vehicle.Passengers.Count

### Expected behaviour

To recheck passangers when player cancel entering task.

### Operating system

Win 11 & Ubuntu 22.04

### Version

release 16.0.158

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest developer version available at the time of writing this ticket.*

**Comment by xLuxy** [comment]
Hey @TuK4z, can you observe the same behavior on JS or is it only related to C# module?

**Comment by TuK4z** [comment]
Currently i cannot, but i can in several days or weeks. If anyone using JS wont do it untill, I ll try to create JS resource and try it.

**Comment by TuK4z** [comment]
I tried in JS, but i don't know JS very well so i couldnt make it to work.

Js have this prototype: IVehiclePassenger and no info in docs how to use it (also no info in dc). In c# i have List<PlayerSeat> to work with.

**Comment by xLuxy** [comment]
Well, the type definition of the IVehiclePassenger interface tells you what it does:

```js
// Vehicle has to bbe obtained somewhere
const passengers = vehicle.passengers;
for (const [seat, player] of Object.entries(passengers)) {
    alt.log(`Seat: ${seat} - Player: ${player.name}`);
}
```

**Comment by TuK4z** [comment]
Yeah but shouldn't it recalculate if task was canceled? Because there is no driver and no passangers in vehicle, but if you use vehicle.Passengers.Count it will be > 0

**Comment by Doxoh** [comment]
its fixed in c# module nuget 16.2.4
